<ui version="4.0" >
 <class>FRWFileRestorePanel</class>
 <widget class="QWidget" name="FRWFileRestorePanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Restore</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>FileType</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Id</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Date</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>231</width>
            <height>161</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton" >
          <property name="text" >
           <string>Restore</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2" >
          <property name="text" >
           <string>Delete from server</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
